hw1/02-palindrome.html

```diff
@@ -6,22 +6,25 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
-    <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
-        </div>
-        <div>
-          <p>Is this a palindrome?</p>
-          <div></div>
-        </div>
-      </section>
-    </main>
+  <body class="bg-primary">
+    <div class="container mt-5">
+      <main class="row justify-content-center">
+        <section class="bg-light col-6 p-4">
+          <h1>Exercise 02 - Palindrome</h1>
+          <div class="mt-3">
+            <label for="num">Enter a positive number:</label>
+            <input type="number" id="num" placeholder="Enter a positive #" />
+          </div>
+          <div class="mt-3">
+            <p>Is this a palindrome?</p>
+            <div class="result mt-3"></div>
```

Comment:
I would have used id='result' here as it is the only item with this class, and it does not have a definition in bootstrap, but that is honestly just being nitpicky.

Reply:
You're right since IDs need to be unique. 

Thanks for your feedback.